compact.go

```diff
@@ -413,6 +427,10 @@ func (c *LeveledCompactor) Write(dest string, b BlockReader, mint, maxt int64, p
 		return uid, err
 	}
 
+	if meta.Stats.NumSamples == 0 {
+		return ulid.ULID{}, nil
```

Comment:
I updated the comments. And about CHANGELOG, I added the `[CHANGE]` info in the PR description so that later when new version will be tagged, it will be easy to add in that.

Reply:
no I meant to actually create a change log file and add it there.